doc/release/release_dev.rst

```diff
@@ -38,6 +38,10 @@ API Changes
   In `is_semiconnected`, the keyword argument `topo_order` has been removed.
   That argument resulted in silently incorrect results more often than not.
 
+- [`#6887 <https://github.com/networkx/networkx/pull/6887>`_]
+  A new `default` argument is added to `get_node_attributes` and
+  `get_edge_attributes`. The `default` keyword can be used to set
+  a default value if the attribute is missing from a node/edge.
```

Comment:
I would think it should be API changes -- since it is changing a function signature. But it is also an improvement. I think we put improvements as things that are better -- but don't have another place to be mentioned.